### DISCUSSION TITLE
Feature Request: Example Documentation

### DISCUSSION THREAD

**Discussion by shenoynikhil** [google/flax#2779] [Ideas] [answered]
In the [API Reference](https://flax.readthedocs.io/en/latest/api_reference/index.html), is it possible to have an example for each module. It's generally easy to understand the usage from an example. 

The request is inspired from pytorch (example shared below for `torch.nn.Conv2D` [Link](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html))
<img width="801" alt="Screenshot 2023-01-08 at 2 29 15 PM" src="https://user-images.githubusercontent.com/28651490/211222324-4d1486c4-71f7-43a4-a316-28eb782560a7.png">

I am also willing to help out to improve the documentation if I can get some support :).

**Comment by IvyZX** [comment, marked as answer]
Thanks for raising it up! @marcvanzee posted it on Issues as a formal feature request :) https://github.com/google/flax/issues/2782
